recipe/meta.yaml

```diff
@@ -31,7 +31,7 @@ source:
 
 build:
   number: 0
-  skip: true  # [osx]
+  skip: true  # [osx or (linux and s390x)]
 
 test:
   commands:
```

Comment:
Should we test on `win` here like it's done [below](https://github.com/AnacondaRecipes/cuda-cccl-feedstock/pull/1/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR70-R71)?

Reply:
sounds good, it's not done on the conda-forge feedstock but I added it here with an informative commit message.